### PR TITLE
CB-71: Add pagination

### PR DESCRIPTION
**Pull request by Narzew**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -15,6 +15,7 @@ gem 'puma', '~> 5.0'
 # gem 'redis', '~> 4.0'
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
+gem 'kaminari'
```
Comment (h4b00) [inline Gemfile:18]: missing version

File: app/controllers/api/v1/recipes_controller.rb [hunk 2]
```diff
@@ -2,9 +2,13 @@ module Api
   module V1
     class RecipesController < ApplicationController
       before_action :set_recipe, except: %i[create index]
+      DEFAULT_PER_PAGE = 24
+
       def index
-        render json: { data: ActiveModel::SerializableResource.new(Recipe.all, each_serialize: RecipeSerializer,
-                                                                               context: { include_comments: false }) }
+        @params_per_page = params[:per_page].to_i
+        @params_per_page = DEFAULT_PER_PAGE if @params_per_page.zero?
```
Comment (h4b00) [inline app/controllers/api/v1/recipes_controller.rb:9]: I would name it `per_page_param`

File: app/controllers/api/v1/recipes_controller.rb [hunk 3]
```diff
@@ -2,9 +2,13 @@ module Api
...
+        render json: { data: ActiveModel::SerializableResource.new(Recipe.page(params[:page]).per(@params_per_page), each_serialize: RecipeSerializer,
+                                                                                                                     context: { include_comments: false }) }
```
Comment (h4b00) [inline app/controllers/api/v1/recipes_controller.rb:11]: you don't have to use this context, it's false by default right?

**Review by h4b00 — CHANGES_REQUESTED** [review]
Missing:
- Response change
- Swagger specs
- New response specs.

File: spec/integration/api/v1/recipes_spec.rb [hunk 4]
```diff
@@ -1,16 +1,26 @@
 require 'swagger_helper'
 RSpec.describe 'Recipes API', :request do
   let(:recipe) { create(:recipe) }
+  let(:page) { rand(1..10) }
+  let(:per_page) { rand(1..50) }
```
Comment (h4b00) [inline spec/integration/api/v1/recipes_spec.rb]:
```suggestion
  let(:page) { 1 }
  let(:per_page) { 3 }
```

**Review by h4b00 — APPROVED** [review]

**Review by mirecki — APPROVED** [review]

**Review by grabomarcin — APPROVED** [review]

**Pull request merged**